src/core/ApplicationDescription.h

```diff
@@ -146,7 +146,7 @@ class ApplicationDescription {
     bool allowAudioCapture() const { return m_allowAudioCapture; }
 
     //Key code is changed only for facebooklogin WebApp
-    const QMap<int, QPair<int, int>>& keyFilterTable() const
+    const std::map<int, std::pair<int, int>>& keyFilterTable() const
```

Comment:
Could you please take a look at the map container usage and if it doesn't need iterating on sorted order then take the opportunity to convert them to unordered_map instead? Please check the same for other occurrences.

Reply:
Ok. I'll take a look at this.
thanks for reviewing it.

Reply:
Done.

Reply:
Done.